src/Parallel/DistributedObject.hpp

```diff
@@ -1009,7 +1009,7 @@ DistributedObject<ParallelComponent, tmpl::list<PhaseDepActionListsPack...>>::
     // profiling is fairly small (<2%), we still avoid it when we aren't
     // tracing.
 #ifdef SPECTRE_CHARM_PROJECTIONS
-    if constexpr (Parallel::is_array_proxy<cproxy_type>::value) {
+    if constexpr (Parallel::is_array<parallel_component>::value) {
       if (not this->thisProxy[array_index_]
                   .template invoke_iterable_action<
```

Comment:
@kidder Not sure. I haven't used tracing/projections in a while. Might have been broken when we changed singletons to be single element charm arrays.  The error occurred when the parallel component was a singleton. The proxy type is an array proxy (because singletons are single element arrays), but our singletons don't have an entry method called `invoke_iterale_action`. This this bug was hit.